src/textarea/textarea.tsx

```diff
@@ -9,25 +9,127 @@ interface Props {
     className?: string;
     placeholder?: string;
     disabled?: boolean;
+    isCharacterLimitEnabled?: boolean;
```

Comment:
I think it might be better to name it like a command. From API view it makes more sense to say `enableCharacterLimit`.